_studio/mfx_lib/shared/src/mfx_h264_encode_vaapi.cpp

```diff
@@ -2733,6 +2800,12 @@ mfxStatus VAAPIEncoder::Execute(
         }
     }
 
+    if (m_caps.RoundingOffset && ctrlRoundingOffset)
+    {
+        MFX_CHECK_WITH_ASSERT(MFX_ERR_NONE == SetRoundingOffset(m_vaDisplay, m_vaContextEncode, *ctrlRoundingOffset, m_roundingOffsetId), MFX_ERR_DEVICE_FAILED);
+
+        configBuffers[buffersCount++] = m_roundingOffsetId;
```

Comment:
are you sure that the size of configBuffer is enough to hold new buffer? I think you need to extend MAX_CONFIG_BUFFERS_COUNT

Reply:
OK